src/components/FoodForm.js

```diff
@@ -1,71 +1,151 @@
 import {StyledForm} from './styled/StyledForm';
+import {StyledImageCenter} from './styled/StyledImage';
 import {StyledInput} from './styled/StyledInput';
+import {StyledAtag} from './styled/StyledLink';
 import {StlyedList} from './styled/StyledList';
-import {StyledListItems} from './styled/StyledListItems';
+import {StyledListItems, StyledFetchItems} from './styled/StyledListItems';
+import {StyledBlackButton, StyledWhiteButton} from './styled/StyledNavButton';
+import {StyledCalories} from './styled/StyledNew';
+
+const StyledFetchButtons = styled.form`
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	margin: 6px;
+`;
+
+const StyledSearch = styled.div`
+	display: flex;
+	justify-content: center;
+	margin: 16px;
+	padding: 16px;
+	border-radius: 16px;
+	background: rgba(0, 0, 0, 0.5);
+	color: white;
+`;
+
+const StyledNutrientList = styled.ul`
+	list-style: none;
+`;
+
+const StyledText = styled.p`
+	color: white;
+	font-family: sans-serif;
+	font-weight: bold;
+`;
+
+const StyledCenterDiv = styled.div`
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	text-align: center;
+`;
 
-//Dummy Data - will be replaced with Data from the API
 const FoodForm = () => {
+	const [data, setData] = useState(null);
+	{
+		console.log(data);
+	}
 	const [calories, setCalories] = useState(0);
 	const [value, setValue] = useState('');
-	const [foods, setFoods] = useState([
-		{
-			id: nanoid(),
-			value: 'Bio-Müsli',
-			done: false,
-			calories: 660,
-		},
-		{
-			id: nanoid(),
-			value: 'Spiegelei',
-			done: false,
-			calories: 420,
-		},
-		{
-			id: nanoid(),
-			value: '1 Banane',
-			done: false,
-			calories: 90,
-		},
-		{
-			id: nanoid(),
-			value: 'Walnüsse',
-			done: false,
-			calories: 140,
-		},
-		{
-			id: nanoid(),
-			value: 'Franzbrötchen',
-			done: false,
-			calories: 375,
-		},
-	]);
+	const [foods, setFoods] = useState([]);
+
+	function FoodSearch(props) {
+		fetch(`/api/food/recipes?query=${props}`)
+			.then(response => response.json())
+			.then(json => setData(json.results));
+	}
+
 	return (
 		<>
 			<StyledForm
 				onSubmit={event => {
 					event.preventDefault();
-					setFoods([...foods, {id: nanoid(), value, done: false}]);
-					setValue('');
+					FoodSearch(value);
 				}}
 			>
-				<StyledDoneLabel>
-					What did you eat today?
+				<StyledSearch>
+					Search for recipes
 					<StyledInput
-						placeholder="e.g. Eggs and bacon"
-						type="text"
-						maxLength="40"
 						value={value}
 						onChange={event => {
+							event.preventDefault();
 							setValue(event.target.value);
 						}}
+						placeholder="e.g. Pasta"
+						type="search"
+						maxLength="40"
 					/>
-				</StyledDoneLabel>
-				<button type="submit">Add</button>
+					<button type="submit">search</button>
+				</StyledSearch>
 			</StyledForm>
+			{data?.map(recipe => {
+				return (
+					<div key={recipe.id}>
```

Comment:
Instead of setting a `div` to be able to hand over the key you can work with `React.Fragment`. [See React docs.](https://reactjs.org/docs/fragments.html#keyed-fragments)